### ISSUE TITLE
Request for removal of finity69x2/nws_alerts

### ISSUE THREAD

**Issue by inguy24** [hacs/integration#3947]
### Repository

finity69x2/nws_alerts

### Checklist

- [X] I understand that this form should only be used for repositories that needs to be removed from HACS
- [X] I understand that a bug is not reason enough to have a repository removed
- [X] The repository is currently shipped as a default repository in HACS
- [X] I have tried to get the authors attention to the reason for removal

### Why should this be removed?

NWS alerts is meant to provide timely and accurate alerts from the National Weather Service on weather phenomena that could be a threat to life and property. The author made numerous changes to the integration that changed the dataset provided by the integration. These changes permanently break existing automations. The author has failed to provide any documentation on the format of the data provided by the integration so automations may be property repaired or the integration being properly used. I asked in Github in no uncertain terms for the author to give an example of how the data is being provided by the integration to allow for individuals to repair and/or create automations properly. The author has FAILED to provide that information. 

As this is not just any data, but data that is important for the timely protection of life and property, I ask that HACS remove this irresponsible integration until such time as the author complies with providing appropriate documentation of the app and dataset that it supposedly provides. 

### Link to issue

https://github.com/finity69x2/nws_alerts/issues/85

**Comment by ludeeus** [comment]
It's a breaking change, that happens.
As others have pointed out in the linked issue, this works as intended when adjusted properly.

**Comment by T3chArmy** [comment]
@inguy24 what the author has done is made a single attribute which contains a standard JSON object with a standardized format. In my opinion this is significantly easier to work with than what was previously implemented with some arbitrary separation. 

With this new format, it's much more consistent and easier to access the information, before you had to do select the field you want to read from the attribute, split it, then select the item from the split you want to read.
```
state_attr('sensor.nws_alerts', 'event_id').split('-')[0]
state_attr('sensor.nws_alerts', 'title').split('-')[0]
```

Now you read the alerts attribute, select the alert number (0, theoretically seems to be the newest issued), and read the named field you want.

```
state_attr('sensor.nws_alerts', 'Alerts')[0].ID
state_attr('sensor.nws_alerts', 'Alerts')[0].Headline
```

As said @ludeeus stated, breaking changes happen. This is why it was released as version 5.0, not version 4.x. This is a standard practice in code development. 

If you're not ready to update versions, no one is forcing you to. The old version still works as of right now.

@finity69x2 updated the [example package](https://github.com/finity69x2/nws_alerts/blob/75cd58ce7f236ef62ca23367cd503b50d6828799/packages/nws_alerts_package.yaml) in the repo. and I also provided [some examples](https://github.com/finity69x2/nws_alerts/issues/87) as well. Between the two of these, it should be relatively simple and straight forward to upgrade your code.
